test/unit/logic/transaction_pool.js

```diff
@@ -911,15 +911,13 @@ describe('transactionPool', () => {
 					null
 				);
 
-			expect(
-				transactionPool.receiveTransactions(transaction, {}, (err, res) => {
-					expect(err).to.eql(
-						'Transaction is already processed: 10431411423423423L'
-					);
-					expect(res).to.deep.eql(transaction);
-					done();
-				})
-			);
+			transactionPool.receiveTransactions(transaction, {}, (err, res) => {
+				expect(err).to.eql(
+					'Transaction is already processed: 10431411423423423L'
+				);
+				expect(res).to.deep.eql(transaction);
```

Comment:
`expect(res).to.deep.equal(transaction);`

Reply:
We don't have standard for this.

Reply:
By reading the documentation, it seems like `deep.eql` doesn't exist

Reply:
I think it is an alias. I'm changing all the matches to `deep.equal` to make it standard. There are less results compare to `deep.equal`.